tests/data/ImportsFlattened.sol.txt

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
+
+// File: @remapping_2_brownie/BrownieContract.sol
+
+pragma solidity ^0.8.4;
+
+contract BrownieContract {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: @styleofbrownie/BrownieStyleDependency.sol
+
+pragma solidity ^0.8.4;
+
+contract BrownieStyleDependency {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: @dependency_remapping/DependencyOfDependency.sol
+
+pragma solidity ^0.8.4;
```

Comment:
I dont think we need more than one pragma in the end.
I think just the one from the root contract should be the one included